diff --git a/src/main/setup/onReadyToShow.ts b/src/main/setup/onReadyToShow.ts
index 17273833..9e5fb174 100644
--- a/src/main/setup/onReadyToShow.ts
+++ b/src/main/setup/onReadyToShow.ts
@@ -6,11 +6,6 @@ const onReadyToShow = () => {
   const window = Window.getWindow();
   window.once("ready-to-show", () => {
     window.show();
-    if (process.platform === "linux") {
-      if (!checkUdev()) {
-        installUdev(window);
-      }
-    }
     autoUpdateOptIn(window);
   });
 };
